components/slices/Process.vue

```diff
@@ -0,0 +1,349 @@
+<template>
+  <section class="process" :style="[background]">
+    <prismic-rich-text
+      v-if="displayTitle && hasTitle"
+      class="process__title"
+      :field="title"
+    />
+    <prismic-rich-text v-else class="sr-only" :field="title" />
+    <prismic-rich-text
+      v-if="useSubtitle && hasSubtitle"
+      class="process__subtitle"
+      :field="subtitle"
+    />
+    <div class="process__wrapper">
+      <div
+        v-for="(itemsInARow, rowIndex) in itemsByRow"
+        :key="`item-${rowIndex}`"
+        class="process-wrapper__row"
+      >
+        <div
+          v-for="(item, itemIndex) in itemsInARow"
+          :key="`item-${itemIndex}`"
+          class="process-wrapper__item"
+        >
+          <pix-image v-if="hasImage(item)" :field="item.item_image" />
+          <div>
+            <prismic-rich-text
+              class="process-wrapper-item__title"
+              :field="item.item_title"
+            />
+            <prismic-rich-text
+              class="process-wrapper-item__description"
+              :field="item.item_description"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import {
+  DESKTOP_MIN_WIDTH,
+  EXTRA_LARGE_SCREEN_MIN_WIDTH,
+  LARGE_SCREEN_MIN_WIDTH,
+  TABLET_MIN_WIDTH,
+} from '~/config/breakpoints'
+import PixImage from '@/components/PixImage'
+
+export default {
+  name: 'ProcessSlice',
+  components: {
+    PixImage,
+  },
+  props: {
+    slice: {
+      type: Object,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      itemsByRow: [],
+    }
+  },
+  computed: {
+    background() {
+      return {
+        backgroundColor: `${this.slice.primary.process_background_color}`,
+      }
+    },
+    items() {
+      return this.slice.items
+    },
+    displayTitle() {
+      return this.slice.primary.process_display_title === 'Yes'
+    },
+    useSubtitle() {
+      return this.slice.primary.process_use_subtitle === 'Yes'
+    },
+    hasTitle() {
+      return (
+        this.slice.primary.process_title.length &&
+        this.slice.primary.process_title[0].text.length
+      )
+    },
+    hasSubtitle() {
+      return (
+        this.slice.primary.process_subtitle.length &&
+        this.slice.primary.process_subtitle[0].text.length
+      )
+    },
+    title() {
+      return this.slice.primary.process_title
+    },
+    subtitle() {
+      return this.slice.primary.process_subtitle
+    },
+  },
+  mounted() {
+    this.itemsByRow = this.splitItemsIntoRows()
+  },
+  methods: {
+    hasImage(item) {
+      return (
+        Object.keys(item.item_image).length &&
+        item.item_image.constructor === Object
+      )
```

Comment:
En lecture rapide, j'ai cru que c'était une méthode
```suggestion
      return Object.keys(item.item_image).length && item.item_image.constructor === Object
```
ou 
```suggestion
      return Object.keys(item.item_image).length &&
        item.item_image.constructor === Object
```
ou
```suggestion
	  const isLengthDifferentThanZero = Object.keys(item.item_image).length
	  const isObjectConstructor = item.item_image.constructor === Object
      return isLengthDifferentThanZero && isObjectConstructor
```